systems/overview/diagrams/03-business-context.drawio.svg

```diff
@@ -0,0 +1,202 @@
+    <defs/>
+    <g>
+        <ellipse cx="584" cy="23.5" rx="7.5" ry="7.5" fill="#dae8fc" stroke="#6c8ebf" pointer-events="all"/>
+        <path d="M 584 31 L 584 56 M 584 36 L 569 36 M 584 36 L 599 36 M 584 56 L 569 76 M 584 56 L 599 76" fill="none" stroke="#6c8ebf" stroke-miterlimit="10" pointer-events="all"/>
+        <g fill="#000000" font-family="Helvetica" text-anchor="middle" font-size="12px">
+            <text x="583.5" y="10.5">
+                Client Editor / Viewer
```

Comment:
```suggestion
                Client Editor
```

Reply:
Whoopsie, thanks for spotting that. Fixed!